src/span.js

```diff
@@ -195,6 +216,20 @@ export default class Span {
         }
```

Comment:
the log() method will likely be different: https://github.com/opentracing/opentracing-javascript/pull/53/files#diff-a334dd7a261db56bf2558d532ee01111R200

we may want to implement the new format, and not require just event/payload to be in the fields.